.github/workflows/release.yml

```diff
@@ -97,6 +101,8 @@ jobs:
           # goreleaser won't sign the packages.
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           SNAPCRAFT_STORE_CREDENTIALS: ${{ secrets.SNAPCRAFT_STORE_TOKEN }}
+          SNAPCRAFT_CHANNEL: ${{ !inputs.prerelease &&  'stable' || 'edge' }}
```

Comment:
It may be nicer here to invert the ternary.

Reply:
Done.